googletest/src/gtest.cc

```diff
@@ -3222,7 +3222,7 @@ static WORD GetNewColor(GTestColor color, WORD old_color_attrs) {
   return new_color;
```

Comment:
Was the "random" key on your keyboard stuck when naming this function?

Reply:
That's wasn't necessary. I choose not to used random